arch/arm/core/aarch32/cortex_m/tz/secure_entry_functions.ld

```diff
@@ -7,14 +7,21 @@
 #if CONFIG_ARM_NSC_REGION_BASE_ADDRESS != 0
 	#define NSC_ALIGN . = ABSOLUTE(CONFIG_ARM_NSC_REGION_BASE_ADDRESS)
 #elif defined(CONFIG_CPU_HAS_NRF_IDAU)
-	/* The nRF9160 needs the NSC region to be at the end of a 32 kB region. */
-	#define NSC_ALIGN . = ALIGN(0x8000) - (1 << LOG2CEIL(__sg_size))
+#ifdef CONFIG_SOC_NRF9160
+#define REGION_SIZE 0x8000
+#elif defined(CONFIG_NRF5340_CPUAPP_ERRATUM19)
```

Comment:
So the logic is:

if 90
else if 53_erratum
else if 53
endif

Would it make more sense to have
if 91
else if 53
if 53_errata
endif 53/91

Not I got by point through, but the errata should only be checked if its a 53

Reply:
Agree. It make sense to only check 53 errata if it's a 53 device (in case it's mixed up)

Reply:
Done